src/app.js

```diff
@@ -1,8 +1,14 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import { BrowserRouter } from 'react-router-dom'
 
-import './styles/base.css'
+import App from '_pages'
```

Comment:
I'm being really picky here, but although we can import a default module by any name, I normally use the same name as it was exported (if any). In this case, does it makes sense to change `App` to `Routes`?

Reply:
I can do that, sure!